client/my-sites/sidebar/publish-menu.jsx

```diff
@@ -78,6 +78,18 @@ const PublishMenu = React.createClass( {
 				buttonLink: site ? '/page/' + site.slug : '/page',
 				wpAdminLink: 'edit.php?post_type=page',
 				showOnAllMySites: true,
+			},
+			{
+				name: 'media',
+				label: this.translate( 'Media' ),
+				className: 'media',
```

Comment:
I suppose it's consistent with other sidebar elements, but these simple class names seem ripe for conflict and don't really serve much purpose in their usage here.